README.md

```diff
@@ -27,7 +27,7 @@ For a good overview and starting point, please check out the official [Solidity
 
 Solidity is a statically-typed curly-braces programming language designed for developing smart contracts
 that run on the Ethereum Virtual Machine. Smart contracts are programs that are executed inside a peer-to-peer
-network where nobody has special authority over the execution, and thus they allow to implement tokens of value,
+network where nobody has special authority over the execution, and thus they allow you to implement tokens of value,
```

Comment:
Agree that it's missing here, but can you maybe change it to `anyone`?

Reply:
Sure